.github/.jira_sync_config.yaml

```diff
@@ -26,7 +26,7 @@ settings:
   add_gh_comment: true
 
   # (Optional) (Default: None) Parent Epic key to link the issue to
-  epic_key: SOLENG-190
+  epic_key: SOLENG-1100
```

Comment:
:( this was another typo... I must be accidentally hitting some key combination in my editor? Or a failed search and replace? Not sure how I did thatk :thinking:

Reply:
Addressing in https://github.com/canonical/solutions-engineering-automation/pull/148